### PR TITLE
Snap 1334 : Auto Refresh feature for Dashboard UI 

### PR DESCRIPTION
**Pull request by snappy-sachin**
## What changes were proposed in this pull request?

- Fixed the JQuery DataTable Sorting Icons problem in the Spark UI by adding missing sort icons and CSS.
- Adding new snappy-commons.js JavaScript for common utility functions used by Snappy Web UI.
- Updated Snappy Dashboard and Member Details JavaScripts for following
  1. Creating and periodically updating JQuery Data Tables for Members, Tables and External Tables tabular lists.
  2. Loading and creating Google Charts.
  3. Creating and periodically updating the Google Line Charts for CPU and various Memory usages.
  4. Preparing and making AJAX calls to snappy specific web services.
  5. Updated/cleanup of Spark UIUtils class.

## How was this patch tested?

 - Manually.

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

File: core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js [hunk 1]
```diff
@@ -111,4 +111,223 @@ function initLogPage(params, logLen, start, end, totLogLen, defaultLen) {
...
+  cpuChartData.addColumn('number', 'CPU');
+
+  var heapChartData = new google.visualization.DataTable();
+  heapChartData.addColumn('datetime', 'Time of Day');
+  heapChartData.addColumn('number', 'JVM');
+  heapChartData.addColumn('number', 'Storage');
+  heapChartData.addColumn('number', 'Execution');
+
+  var offHeapChartData = new google.visualization.DataTable();
+  offHeapChartData.addColumn('datetime', 'Time of Day');
```
Comment (rishitesh) [inline core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js:171]: a header OFF-HEAP not required here?

**Review by rishitesh — APPROVED** [review]
One minor comment.

File: core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js (hunk 1 above)
Comment (snappy-sachin) [reply core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js:171]: Header is specified in chart configuration options.

**Comment by snappy-sachin** [comment]
Merging.

**Pull request merged**